src/notation/internal/notationactioncontroller.cpp

```diff
@@ -1244,6 +1244,8 @@ void NotationActionController::startEditSelectedElement(const ActionData& args)
     if (interaction->textEditingAllowed(element)) {
         PointF cursorPos = !args.empty() ? args.arg<PointF>(0) : PointF();
         interaction->startEditText(element, cursorPos);
+    } else if (element->hasGrips()) {
```

Comment:
but an Image *does* have grips, surprised that this makes a difference...

Reply:
This code is needed for the "Edit element" item in the context menu. It automatically selects the default grip point of an element and starts "edit mode"

Reply:
Ok but I believe if you use that mode then you *can't* use the mouse to move an image.  But that's the same as v3.